10x11 Befenybay Drag Chain Top Anchor and End Mount to Extrusion Plus Custom Slicer Bed Profiles

bryansj opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
10x11_Befenybay_Chain_Top_Anchor
#### How was it tested? 
Printed and installed
#### Any background context you want to provide?
Modified Manfre's similar part to fit the 10x11 Befenybay Drag Chain on Amazon.  Needed hole pins and width resized.
#### Screenshots (if appropriate)

#### Further notes
Requested to be uploaded by Eddie

bryansj commented:
I've added in slicer_configurations for my bed models and for any future slicer profiles.

bryansj commented:
Added another aftermarket drag chain mount (mount to 2020 extrusion).

FHeilmann commented on printer_mods/bryansj/Triangle_Hole_Pattern_End_Mount_to_2020/readme.md:
```diff
@@ -0,0 +1,11 @@
+
+These are to replace the similar parts of the Zip Chain installation.
+
+To install use a M6 BHCS, T-nut, and zip-tie.
```
I think you mean M3x6 here right?

bryansj replied:
I meant M3x6...

bryansj commented:
> Hi, thanks for your mod, a few comments:
> 
> * You have two mods which both relate to the befenybay chains. I think these should be one mod, since anyone who has these chains will probably want all those files anyway. Please merge them into one mod.

I merged them into one.  They were separate because I did one and then a few days later made the other.

> * Your PR modifies eddies config files. That should not happen.

That was an accident from updating the wrong tab in VSCode.  I actually reverted it before I got your message.  That shouldn't have happened.

> * Except for the case-light stuff and some changed currents I don't see many changes in your proposed config. Can you elaborate a little on why your config should coexist with the config by eddie which it is based on? Right now I don't see much value in having a config that is almost equal. You already have some additional snippets submitted to this repo. Wouldn't it make sense if you just added your new additions to that?

I just added my config since it adds stuff frequently requested and stuff that I've sought out from the community and added.  I'm fine if you would like to remove it.  This is what is added after you get Eddie's config running.  It is not meant to be used instead of his config, but to add to extended it (lighting/nozzle wipe/prime line, chamber fan).  I tried to document this at the beginning, but I'm sure people won't always read it.

#   Nozzle Wipe
#       added to G32
#   Prime Line
#   Extended QGL corner points
#   Bed Meshing (Commented Out)
#   TL Filament Run-out Sensor
#   Chamber Temperature Thermistor (in Z vertical Chain)
#       linked to Back Panel Exhaust fan 
#   Force Move Enabled
#   Pause/Resume Enabled
#   Homing IF/THEN to omit unneeded homing operations
#   M117 LCD Text Added *16 Characters Max
#   Expanded LCD Menu Options for Voron/Klipper Commands
#       Quad Gantry Level, Firmware_Restart, TESTZ Offsetting
#   LED Lighting Control


> * A small comment in your readmes regarding the mounting of the chains.
> 
Fixed the 
> Other than that the PR looks good. Thanks for taking the time to create the bed models.
> 
> -Florian
